application-provider/src/main/java/org/jboss/intersmash/applications/ApplicationProvider.java

```diff
@@ -57,6 +58,18 @@ public static Path wildflyDistributedSessionsInfinispanProvisionedServerPath() {
 		return findApplicationDirectory("wildfly", "distributed-sessions-infinispan", "target", "server");
 	}
 
+	/**
+	 * Provides access to a filesystem directory containing a server provisioned by
+	 * the WildFly/JBoss EAP 8.x Maven plugin, which is generated from the
+	 * {@code wildfly-amq-broker-ssl} application
+	 *
+	 * @return {@link Path} instance that identifies the directory containing the
+	 *         WildFly/JBoss EAP 8.x provisioned server.
+	 */
+	public static Path wildflyActiveMQArtemisProvisionedServerPath() {
```

Comment:
rename `wildflyActiveMQArtemisProvisionedServerPath` to `wildflyActiveMQArtemisSslProvisionedServerPath`?